.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
Personally, I like the option of having a blank issue as there are various categories of "issues" that might not be covered by `bug` and `feature`.